cmd/importer/pod/import.go

```diff
@@ -206,7 +215,9 @@ func admitWorkload(ctx context.Context, c client.Client, wl *kueue.Workload, cq
 		return wl, true, nil
 	}
 
-	err := workload.PatchAdmissionStatus(ctx, c, wl, realClock, update)
+	err := clientutil.PatchStatus(ctx, c, wl, func() (client.Object, bool, error) {
```

Comment:
Why this changes?

Reply:
We need to operate using v1beta1 as this code is running while Kueue is disabled.

It does not really matter if we use Patch or Apply while Kueue is at sleep.

Reply:
I think it does, because we need a field owner to set some fields to nil.